docs/developer/windows/certificate_store/README.md

```diff
@@ -0,0 +1,143 @@
+
+7. Select `Root CA`.
+![](./images/root_ca.png)
+
+8. Select `Create a new private key`.
+![](./images/private_key.png)
+
+9. Go with the defaults.
+![](./images/default_private_ky.png)
+
+10. Go with validity of 5 years.
+![](./images/validity.png)
+
+11. Install.
+12. When asked if you want to install additional services say yes.
+![](./images/additional_services.png)
+
+13. Select `Certificate Enrollment Web Service`.
+14. Use the default application pool identity.
+![](./images/default_identity.png)
+
+15. Choose existing SSL certificate.
+![](./images/default_certificate.png)
+
+16. Click through and complete.
+17. Reboot.
+
+## Creating a template
+
+1. From Server Management click Tools->Certificate Authority.
+![](./images/certificate_authority_click.png)
+
+2. Ensure `Certificate Templates` is available, if not then you are not set up as an Enterprise CA and go back to `Setup as domain controller`.
+![](./images/certificate_templates.png)
+
+3. Right-click on `Certificate Templates` and select `Manage`.
+![](./images/manage.png)
+
+4. This opens the `Certificate Templates Console`.
+5. Select `Authenticate Session` and `Duplicate`.
+6. Open `General` tab and change names to `Certstore Template`, set the Renewal Period to 32850 hours that is the max allowed.
+![](./images/template_general.png)
+
+7. Open `Cryptography` tab and select `Request can use any provider available on the subject's computer`.
+![](./images/availability.png)
+
+8. Open `Request Handling` tab and select `Allow private key to be exported`.
+![](./images/export_private_key.png)
+
+9. Open `Security` tab and give Administrator all the control.
+![](./images/give_control.png)
+
+10. Open `Extensions` tab click `Application Policies` click `Edit`, ensure `Any Purpose` is selected.
+![](./images/any_purpose.png)
+
+11. Close `Certificate Template Console`.
+12. Reopen `Certificate Authority` console.
+13. Right-click on `Certificate Templates`, select `New`, then `Certificate Template to Issue`.
+14. Select `Certstore Template`.
+15. It will now show in the Templates folder of Certificate Authority.
+
+## Creating a certificate from the template
+
+1. Open `certmgr` also called `Manage user certificates` from the Run button.
+2. Expand `Personal`, right-click on personal, select `All Tasks` , select `Request New Certificate`.
+3. Click through until you get the `Request Certificate` screen and select `Certstore Template`.
+![](./images/new_cert.png)
+
+4. Expand `Certstore Template` and select `Properties`.
+5. Give it a name  like `TestAuth` and ensure under `Private Key` -> `Key options` that the private key is exportable.
+![](./images/new_cert_exportable.png)
+
+6. Create Certificate.
+
+## Export the certificate
+
+1. From `certmgr` find the certificate you created and double click to open.
+2. Go to the `Details` tab and select `Copy to File...`.
+3. Ensure that `Yes, export the private key` is selected.
+![](./images/export_private_key.png)
```

Comment:
Did you mean "Allow private key to be exported" instead of "Yes, export the private key"? Or is this the wrong screenshot?

Reply:
Good catch!